src/plugins/chatview/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @module converse-chatview
+ * @copyright 2020, the Converse.js contributors
+ * @license Mozilla Public License (MPLv2)
+ */
+import { __ } from '@converse/headless/i18n';
+import { _converse, api, converse } from "@converse/headless/converse-core";
+
+// Plugins
```

Comment:
I think once the plugins are all moved into the plugins folder, the nature of these imports will be clearer, and comments such as this won't be necessary. It's IMO already unnecessary for the components being imported below.

Reply:
I don't think these comments are necessarily keepers in the long term, just used them here as a way to make the import categories super obvious, but they don't need to make it to production.